### ISSUE TITLE
Add DEM to Sen1Floods11

### ISSUE THREAD

**Issue by jamesmcclain** [azavea/noaa-flood-mapping#51]
Add DEM data to the Sen1Floods11 dataset and look for a detectable difference (however small) in model performance.  If detected, this is most likely sufficient (but not necessary) to conclude that HAND will provide value.

**Comment by jamesmcclain** [comment]
### Permanent ###
```bash
python3 /workdir/sen1floods11_to_tiffs.py --imagery-dir /data/S1Perm/ --elevation-dir /data/S1Perm-elevation/ --flood-label-dir /data/PermJRC/ --imagery-file /data/tiffs/with_DEM/permanent/imagery.tif --labels-file /data/tiffs/with_DEM/permanent/labels.tif
```

### Weak Sentinel-2 ###
```bash
python3 /workdir/sen1floods11_to_tiffs.py --imagery-dir /data/S1_NoQC/ --elevation-dir /data/S1_NoQC-elevation/ --flood-label-dir /data/NoQC/ --imagery-file /data/tiffs/with_DEM/weak-s2/imagery.tif --labels-file /data/tiffs/with_DEM/weak-s2/label
s.tif
```

### Weak Sentinel-1 ###
```bash
python3 /workdir/sen1floods11_to_tiffs.py --imagery-dir /data/S1_NoQC/ --elevation-dir /data/S1_NoQC-elevation/ --flood-label-dir /data/S1Flood_NoQC/ --imagery-file /data/tiffs/with_DEM/weak-s1/imagery.tif --labels-file /data/tiffs/with_DEM/weak-s1/labels.tif
```

### Hand-Labeled Training Set ###
```bash
python3 /workdir/sen1floods11_to_tiffs.py --imagery-dir /data/S1/ --elevation-dir /data/S1-elevation/ --flood-label-dir /data/QC_v2/ --perm-label-dir /data/Perm/ --mode train --test-set-csv /data/flood_test_data.csv --imagery-file /data/tiffs/with_DEM/hand/train/imagery.tif --labels-file /data/tiffs/with_DEM/hand/train/labels.tif
```

### Hand-Labeled Test Set ###
```bash
python3 /workdir/sen1floods11_to_tiffs.py --imagery-dir /data/S1/ --elevation-dir /data/S1-elevation/ --flood-label-dir /data/QC_v2/ --perm-label-dir /data/Perm/ --mode test --test-set-csv /data/flood_test_data.csv --imagery-file /data/tiffs/with_DEM/hand/test/imagery.tif --labels-file /data/tiffs/with_DEM/hand/test/labels.tif
```

### Hand-Labeled Bolivia Set ###
```bash
python3 /workdir/sen1floods11_to_tiffs.py --imagery-dir /data/S1/ --elevation-dir /data/S1-elevation/ --flood-label-dir /data/QC_v2/ --perm-label-dir /data/Perm/ --mode test --test-set-csv /data/flood_bolivia_data.csv --imagery-file /data/tiffs/with_DEM/hand/bolivia/imagery.tif --labels-file /data/tiffs/with_DEM/hand/bolivia/labels.tif
```

**Comment by jamesmcclain** [comment]
Description/Bolivia | hash | recall | precision | f1 | iou
-- | -- | -- | -- | -- | --
weak s1 w/o DEM (epochs2 == 13) | cc35f12eabf716298a1a34011b0d041561409baaaafc29a55622611390ccaf5d | 0.8032940501 | 0.5845595914 | 0.6766898579 | 0.511361499
all w/o DEM (epochs2 == 0) | b0b7f1db15168b534dcb7c701e8c4adf9bcb975e7f7b00a7b4092478a2ac4a38 | 0.6133163369 | 0.6608721755 | 0.636206806 | 0.4664980063
weak s1 w/ DEM (epochs2 == 13) | a3735279d0501a35187100864040814888180f7b2971b870e084a9987a67d3f5 | 0.7942161365 | 0.5968095612 | 0.6815054311 | 0.5168814853
all w/ DEM (epochs2 == 0) | 1285faf816e6730c5aac219fc7fc7081a33e4c809f6e607734e498c4b1b4f3de | 0.747830418 | 0.617261154 | 0.6763013944 | 0.5109179624



Description/Test | hash | recall | precision | f1 | iou
-- | -- | -- | -- | -- | --
weak s1 w/o DEM (epochs2 == 13) | cc35f12eabf716298a1a34011b0d041561409baaaafc29a55622611390ccaf5d | 0.736211866 | 0.6904297908 | 0.7125862367 | 0.5535021117
all w/o DEM (epochs2 == 0) | b0b7f1db15168b534dcb7c701e8c4adf9bcb975e7f7b00a7b4092478a2ac4a38 | 0.7191175916 | 0.7576985564 | 0.7379041213 | 0.5846656611
weak s1 w/ DEM (epochs2 == 13) | a3735279d0501a35187100864040814888180f7b2971b870e084a9987a67d3f5 | 0.7400522654 | 0.7097156672 | 0.7245665675 | 0.5680943819
all w/ DEM (epochs2 == 0) | 1285faf816e6730c5aac219fc7fc7081a33e4c809f6e607734e498c4b1b4f3de | 0.7305849823 | 0.7504079057 | 0.7403637802 | 0.5877599965
